src/Cake.Common/Build/BuildSystem.cs

```diff
@@ -581,6 +591,20 @@ public sealed class BuildSystem
         /// </example>
         public ITFBuildProvider TFBuild { get; }
 
+        /// <summary>
```

Comment:
Also in this file:
```csharp
public bool IsRunningOnAzurePipelines => TFBuild.IsRunningOnAzurePipelines;
```
and
```csharp
public bool IsRunningOnAzurePipelinesHosted => TFBuild.IsRunningOnAzurePipelinesHosted;
```

How to solve that? Create another property or replace or add || or .... ?

Reply:
These should now call to `AzurePipelines`